### ISSUE TITLE
Sony loaded only 19 questions in stats section

### ISSUE THREAD

**Issue by yw666666** [rih28/MBAndroidTest#13]
It seems that the app didn’t realise it had a missing question... I reinstalled it and then it is fine now.

**Comment by rih28** [comment]
I will fix this one tonight. It is a pain this condition in the app.

**Comment by rih28** [comment]
It would be good to know if you encountered this again? I am shipping a new version tonight........hopefully

**Comment by yw666666** [comment]
In this newest version, both phones didn't download all the questions while they looked like they downloaded all of them. eg Sony had 15 questions in stats while Huawei had 18.

And the "next Q" button seems not to be working well. I kept clicking on the next button to count the number of questions, but after 5-6 clicks, I had to click a few times to get to the next question and it became quite annoying. I am not sure if this is the problem of the newest version or it's the problem of the phones.

**Comment by rih28** [comment]
Yes, it seems to be that there is a problem with this when not all questions have been downloaded. I will try again with this but likely after you finish tomorrow. Study should be up so I would try and download all questions until you manage it.
